docs/admin/deployments.rst

```diff
@@ -611,15 +609,15 @@ with the following command:
     rhc -aweblate app create -t python-2.7 --from-code https://github.com/WeblateOrg/weblate.git#weblate-2.10 --no-git
```

Comment:
This seems oddly outdated.

Reply:
Fixed in 0daea937a1 and automated update on release in 302326604c, thanks for pointing out.